### PR TITLE
Casting ambiguous syntax

### LINKED ISSUE #191: ambiguous syntax (function call x new statement) near '('
```
(this.something as any).Initialize();
```

=>

```
(self.something ):Initialize()
```

Throws `ambiguous syntax (function call x new statement) near '('` at lua runtime.

### PR DESCRIPTION
**Pull request by andreiradu**
using extra paranthesis in ts causes code like `dostuff() (self.member):method();` calls, which can be ambiguous depending on context. I've added ; at the end of statements to help disambiguate. 

Closes #191

### CODE REVIEW

File: src/Transpiler.ts [hunk 1]
```diff
@@ -1421,9 +1421,9 @@ export abstract class LuaTranspiler {
             const identifierName = this.transpileIdentifier(node.name);
             if (node.initializer) {
                 const value = this.transpileExpression(node.initializer);
-                return `local ${identifierName} = ${value}\n`;
+                return `local ${identifierName} = ${value};\n`;
```
Comment (Perryvw) [inline src/Transpiler.ts]: You pushed these too far down, I suggest adding the ';' in the `transpileNode` function, like you did for ExpressionStatement.

Comment (andreiradu) [reply src/Transpiler.ts]: That was my first approach, but `transpileVariableDeclaration` returns a newline, so the transpiled code would look like
```
      local a = 0
;
      foo()
```

Comment (Perryvw) [reply src/Transpiler.ts]: I think that's a bug, if removing the `\n` from transpileVariableDeclaration does not break any (non-whitespace) tests I am fine with you removing it there.

Comment (lolleko) [reply src/Transpiler.ts]: That new line after variable declaration is definitely unintentional and should be removed.

File: src/Transpiler.ts [hunk 2]
```diff
@@ -497,7 +497,7 @@ export abstract class LuaTranspiler {
         // Add header
         let result = "";
         for (const variableDeclaration of (node.initializer as ts.VariableDeclarationList).declarations) {
-            result += this.indent + this.transpileVariableDeclaration(variableDeclaration);
+            result += this.indent + this.transpileVariableDeclaration(variableDeclaration) + "\n";
```
Comment (Perryvw) [inline src/Transpiler.ts]: Should there also be a `;` here?

Comment (andreiradu) [reply src/Transpiler.ts]: Not really, you know a while will always follow, so there's no ambiguity. I've tried not to add ; where they weren't needed.

**Comment by andreiradu** [comment]
Merged master and fixed tests

**Pull request merged**